### PR TITLE
Add flipud and fliplr numpy functions.

### PR DESCRIPTION
**Pull request by alexalemi**

### CODE REVIEW

File: jax/numpy/lax_numpy.py [hunk 1]
```diff
@@ -493,6 +493,16 @@ def flip(m, axis):
   return lax.rev(m, [axis])
 
 
+@_wraps(onp.fliplr)
+def fliplr(m):
+  return flip(m, 1)
```
Comment (hawkinsp) [inline jax/numpy/lax_numpy.py]: What if the array has rank > 2? I think these need to be phrased as -1, -2.
However, I think this also points to a bug in the implementation of `flip` — it needs to support negative axes.

https://docs.scipy.org/doc/numpy-1.15.1/reference/generated/numpy.fliplr.html

You could also write these two as:
`m[..., ::-1]`
and
`m[..., ::-1, :]`

but the problem with `flip` still remains.

Comment (alexalemi) [reply jax/numpy/lax_numpy.py]: Numpy flipud works on the first axis, not the last: https://github.com/numpy/numpy/blob/v1.15.1/numpy/lib/twodim_base.py#L137

Comment (hawkinsp) [reply jax/numpy/lax_numpy.py]: Huh. You're right. The docs are actually a bit misleading.

The problem with `flip` not accepting negative axes is real.

Please add a 3D test.

Comment (alexalemi) [reply jax/numpy/lax_numpy.py]: Added support for negative indices, but when I tried to support None, ran into deeper issues.  `None` behavior and setting it as a default was just added in numpy 1.15.  Presumably its alright to leave that off?

**Comment by hawkinsp** [comment]
Looks good! Thanks for the PR!

I think it's fine to not support `None` axes yet. We can improve the implementation if someone needs it.

(It looks like `flip` also supports tuples of axes, but let's save that for a future PR.)

**Pull request merged**